### ISSUE TITLE
IOS Hung On Start After Package Version Bumped to 2.1.6

### ISSUE THREAD

**Issue by AndrewKahr** [PlayEveryWare/eos_plugin_for_unity#163]
**Describe the bug**
When the development branch is built for iOS, launching the app results in it being hung on the initial screen with none of the buttons being functional.

**To Reproduce**
1. Pull latest development branch
2. Build UPM package
3. Import into project
4. Build for iOS
5. See the hung application

**Expected behavior**
Log should populate with messages and buttons should respond

**Smartphone (please complete the following information):**
 - Device: iPhone13 Pro
 - OS: iOS 15.7.1
 - Unity Version: 2022.2.0b10
 - Plugin Version: Development

**Additional context**
I walked back the commit history to see where it initially stopped working and interestingly, it doesn't seem to be any direct code changes. The code works up until d7c89a0bf9a1a0853785587d0b4dac2da7207b77 but once you use a0468b608a2b57f3ae5a55fe3fdbc9752e432b0b or later, the app starts hung on boot. It seems that changing the package.json version number back to 2.1.5 fixes the hang. I saw in https://github.com/PlayEveryWare/eos_plugin_for_unity/blob/c92480014950fda281f42939aa1c8c611781ac13/Assets/Plugins/Source/DLLHandle.cs#L55 that there's a hardcoded version number but bumping it up to 2.1.6 didn't fix the hang. Are there other locations with hardcoded version numbers?

**Comment by arthur740212** [comment]
Hi, thanks for the feedback. We are working on a fix.
In the meantime, as we weren't able to reproduce the exact error, we did find a working setting.
While the fix might not be up immediately, here are some potential fixes for you to try out.
1. Add EOS_PREVIEW_PLATFORM to Scripting Define Symbols in Player Settings
2. Check if eos_ios_config.json and EpicOnlineServicesConfig.json exist in Assets/StreamingAssets.

**Comment by AndrewKahr** [comment]
Checked to make sure the above was true but it still was hanging so I created a new project, reimported everything, and now it seems to be running fine. I'm guessing something odd got cached which was breaking it. 
It also seems that the hardcoded version string only affects the version number reported in the prefab as 2.1.6 is still noted as 2.1.5 when running the scene, but other than that, it seems to be fine now.
